start.sh

```diff
@@ -40,8 +43,63 @@ function start_standard {
       nodeos --genesis-json $NODEOS_HOME/config/genesis.json
 }
 
-if [ -d "$HOST_WAX_HOME/nodeos/data/state" ]; then
-  start_standard
+function start_ship_nodeos_from_snapshot {
+  mkdir -p $HOST_WAX_HOME/shipnodeos/data/
+  cd $HOST_WAX_HOME/shipnodeos/data/
+  rm *.bin
+  rm $FILE_NAME
+  wget $MAINNET_SNAPHOT$FILE_NAME
+  unzstd $FILE_NAME
+  rm $FILE_NAME
+  TGZ_FILES=( *.bin )
+  SNAPSHOT="${TGZ_FILES[0]}"
+
+  docker run -t --sig-proxy=true --name nodeos \
+      -v $HOST_WAX_HOME/shipnodeos/data:$NODEOS_HOME/data \
+      -v $HOST_WAX_HOME/shipnodeos/config:$NODEOS_HOME/config \
+      -p 0.0.0.0:8888:8888 \
+      -p 0.0.0.0:8080:8080 \
+      -p 9876:9876 \
+      waxteam/waxnode:$WAX_VERSION \
+      nodeos --verbose-http-errors --disable-replay-opts --trace-history --chain-state-history --snapshot $NODEOS_HOME/data/$SNAPSHOT
+}
+
+function start_ship_nodeos {
+  docker run -t --sig-proxy=true --name nodeos \
+      -v $HOST_WAX_HOME/shipnodeos/data:$NODEOS_HOME/data \
+      -v $HOST_WAX_HOME/shipnodeos/config:$NODEOS_HOME/config \
+      -p 0.0.0.0:8888:8888 \
+      -p 0.0.0.0:8080:8080 \
+      -p 9876:9876 \
+      waxteam/waxnode:$WAX_VERSION \
+      nodeos --disable-replay-opts --trace-history --chain-state-history --genesis-json $NODEOS_HOME/config/genesis.json
+}
+
+while getopts s:e: flag
+do
+    case "${flag}" in
+        s) START_FROM_SNAPSHOT=${OPTARG};;
+        e) ENABLE_SHIP_NODE=${OPTARG};;
+        ?)
+          echo "script usage: $(basename $0) [-s true/false] [-e true/false]" >&2
+          echo "options:"
+          echo "-s     Start From Snapshot."
+          echo "-e     Enable Ship Node."
+          exit 1
+          ;;
+    esac
+done
+
+if [[ $START_FROM_SNAPSHOT == true ]]; then
```

Comment:
We should also check whether the snapshot has already been used by checking if the data/state folder exists. See here: https://github.com/worldwide-asset-exchange/wax-node/blob/master/start.sh#L43
This way we don't re-run the snapshot logic every time someone restarts the service or when it reboots.
Remember that the way to "reset" the system so as to try a new snapshot is to first run the `reset.sh` script: https://github.com/worldwide-asset-exchange/wax-node/blob/master/reset.sh and then start the service again